std/meta.d

```diff
@@ -1120,42 +1120,22 @@ template templateOr(Preds...)
     }
 }
 
+import std.traits : isIterable;
+
 /**
- * Converts an input range `range` to an alias sequence.
+ * Converts any foreach-iterable (e.g. an input range) $(D range) to an alias sequence.
```

Comment:
I assume you accidentally converted the backticks back to `$(D`?

Reply:
Must have a been a mistake when rebasing. Also, it should have been `iter` anyway